tox.ini

```diff
@@ -43,6 +43,7 @@ setenv =
     MOLECULE_NO_LOG={env:MOLECULE_NO_LOG:0}
     PIP_DISABLE_PIP_VERSION_CHECK=1
     PY_COLORS={env:PY_COLORS:1}
+    PYTEST_CHECK_TEST_DUPLICATE=0
```

Comment:
The duplicate name has been fixed in #197.

Could you please remove this item?